File: EventSphere/ViewModels/EventDto.cs
public class EventDto
{
    public int EventId { get; set; }
    public int OrganizationId { get; set; }
    public string Name { get; set; } = null!;
    public DateTime StartDateTime { get; set; }
    public DateTime EndDateTime { get; set; }
    public int EventTypeId { get; set; }
    public int EventStatusId { get; set; }
    public int? OrganizerUserId { get; set; }
    public int? MaxAttendees { get; set; }
    public bool IsPublic { get; set; }
    public DateTime? RegistrationDeadline { get; set; }
    public string? ImageUrl { get; set; }  
    public string? Description { get; set; }
    public string? Location { get; set; }
    public string? City { get; set; }
    public string? Country { get; set; }
}

File: frontend/src/types/EventDto.ts
export interface EventDto {
    eventId?: number; // create işlemi sırasında olmayabilir
    organizationId: number;
    name: string;
    startDateTime: string;
    endDateTime: string;
    eventTypeId: number;
    eventStatusId: number;
    organizerUserId?: number;
    maxAttendees?: number;
    isPublic: boolean;
    registrationDeadline?: string;
    addressId: number
    location?: string; // sadece GET işlemlerinde geliyor olabilir
    imageUrl?: File | string;
    description?: string;
    city: string; // sadece GET işlemlerinde geliyor olabilir
    organizationName?: string;
    country: string; // sadece GET işlemlerinde geliyor olabilir
}
